sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-calculation-process.ts

```diff
@@ -152,13 +170,56 @@ export abstract class ECertCalculationProcess {
     }
   }
 
+  /**
+   * Checks whether the disbursement blocked notification should be created or not.
+   * @param disbursementId disbursement id.
+   * @param entityManager entity manager to execute in transaction.
+   * @returns boolean indicating if the disbursement blocked notification should be created or not.
+   */
+  private async shouldCreateDisbursementBlockedNotification(
```

Comment:
The `ECertCalculationProcess` was created with the idea of keeping a centralized place for the calculation only and it grew too much because of the notifications.
I would suggest creating an `e-cert-notification-services` to keep all notification-related method now in this class.